fabtests/multinode/include/core.h

```diff
@@ -106,6 +116,28 @@ static inline int timer_index(int iter, int dest_rank)
     return iter * pm_job.num_ranks + dest_rank;
 }
 
+#ifndef _WIN32
+static inline void log_print(FILE *f, char *fmt, ...)
+{
+	va_list args;
+
+	if (!pm_job.clients)
+		return;
+	va_start(args, fmt);
+	vfprintf(f, fmt, args);
+	va_end(args);
+}
+
+#define PRINTF(fmt, args...) log_print(stdout, fmt, ## args)
+#define EPRINTF(fmt, args...) log_print(stderr, fmt, ## args)
+
+#else
+
+#define PRINTF(fmt, args...) fprintf(stdout, fmt, ## args)
+#define EPRINTF(fmt, args...) fprintf(stderr, fmt, ## args)
```

Comment:
MSVC doesn't like this definition and that's why the appveyor tests are failing.

Standard variadic macro looks like this:
```
#define MACRO(s, ...) printf(s, __VA_ARGS__)
```